src/Demos/HSP.lagda

```diff
@@ -444,10 +383,8 @@ is a pair \ab{𝑆} = \AgdaPair{F}{ρ} where \ab{F} is a collection of
 \defn{operation symbols} and \ab{ρ} : \ab{F} \as{→} \ab{N} is an \defn{arity function}
 which maps each operation symbol to its arity. Here, \ab{N} denotes the \emph{arity type}.
 Heuristically, the arity \ab{ρ} \ab{f} of an operation symbol \ab{f} \as{∈} \ab{F} may be
-thought of as the number of arguments that \ab{f} takes as ``input.''
-
-The \agdaalgebras library represents an algebraic signature as an
-inhabitant of the following dependent pair type:
+thought of as the number of arguments that \ab{f} takes as ``input.''. This can be
+represented as follows:
```

Comment:
It may seem pedantic at first, but I want to change "This can be represented as follows:" to either:

"We represent this in the agda-algebras library as follows:"

OR

"The \agdalagebras library represents a signature as an inhabitant of the following dependent pair type:"

I think the latter is much clearer and more precise, but either is okay with me.

The reason I'm pushing for this change is that I want to emphasize that this is how *we* represent signatures in type theory.  Apparently, it's typically done differently.  This is meant to be a "selling point" (albeit a minor one) of the agda-algebras library. :)

Reply:
I'll have to think about this. I don't disagree about the point, just how to present it.  In particular, it is common (now) in CS papers to have an explicit paragraph in the introduction outlining the contributions of the paper (which includes the contributions of the software being described)

Reply:
Ah, okay, good point.  I'll let you think.  :)